packages/app/error-utils/package.json

```diff
@@ -25,12 +25,12 @@
         "@bugsnag/js": "^8.0.0"
     },
     "peerDependencies": {
-        "@lokalise/node-core": ">=11.0.0"
+        "@lokalise/node-core": ">=13.0.0"
```

Comment:
do we need to narrow it down?

Reply:
Same as with fastify-extras 😅, will revert it

Reply:
Change reverted